redash/utils/parameterized_query.py

```diff
@@ -1,9 +1,29 @@
 import pystache
+from flask_login import current_user
 from numbers import Number
-from redash.utils import mustache_render
+from redash import models
+from redash.utils import mustache_render, json_loads
+from redash.permissions import require_access, view_only
 from funcy import distinct
 from dateutil.parser import parse
 
+
+def dropdown_values(query_id, org):
+    def _pluck_name_and_value(row):
```

Comment:
I would move this function to be at the module level. Defining functions inside functions always felt dirty to me (not that I haven't done it...).

Reply:
Lambdas make sense if it's a short logic (one line), which in this case it isn't.

I don't see a reason to place functions in the scope of another function aside for the purpose of closures.

Reply:
I don't mind taking this out to a private module function, but I'm still against testing it directly 😛

Reply:
I'll take this :)